operator/helper/output.go

```diff
@@ -65,10 +65,23 @@ func (o *OutputOperator) Outputs() []operator.Operator {
 	return []operator.Operator{}
 }
 
+// Outputs will always return an empty array for an output ID.
+func (o *OutputOperator) GetOutputIDs() []string {
+	return []string{}
+}
+
 // SetOutputs will return an error if called.
 func (o *OutputOperator) SetOutputs(operators []operator.Operator) error {
 	return errors.NewError(
 		"Operator can not output, but is attempting to set an output.",
 		"This is an unexpected internal error. Please submit a bug/issue.",
 	)
 }
+
+// SetOutputs will return an error if called.
+func (o *OutputOperator) SetOutputIDs(opIDs []string) error {
```

Comment:
I believe this is the only circumstance where an actual error is returned, so we can probably just remove the return value from this method signature and make this a do nothing.

Reply:
Removed error from the signature.